rkt/api_service.go

```diff
@@ -395,6 +395,16 @@ func (s *v1AlphaAPIServer) ListPods(ctx context.Context, request *v1alpha.ListPo
 	if err := walkPods(includeMostDirs, func(p *pod) {
 		pod, manifest := getBasicPod(p)
 
+		flavor, err := os.Readlink(filepath.Join(p.path(), "/stage1/rootfs/flavor"))
```

Comment:
You could probably load the stage1 manifest at "stage1/manifest" and check the field "name" in the json file:
```
   "name" : "coreos.com/rkt/stage1-coreos",
```